custom_components/ha_hatch/__init__.py

```diff
@@ -73,7 +73,7 @@ def _install_alpine_dependencies():
 
 def _lazy_install():
     _install_alpine_dependencies()
-    custom_required_packages = ["hatch-rest-api==1.17.0"]
+    custom_required_packages = ["hatch-rest-api==1.20.0"]
```

Comment:
1.19.1 not 1.20.0, fixes increase the last number, new features increase the second number (feat)

Reply:
This has been changed.